src/config/vcap-services.js

```diff
@@ -9,17 +9,25 @@ function getVCAPServices (service) {
   return services
 }
 
-function getVariableService (userProvidedServices) {
-  const variableServices = userProvidedServices.filter(service => service.instance_name === 'variable-service')
+function getService (userProvidedServices, serviceName) {
+  const variableServices = userProvidedServices.filter(service => service.instance_name === serviceName)
```

Comment:
Should this variable just be `services` to match the updated function name?

Reply:
Updated to services.